File: day01.cpp
#include <iostream>
using namespace std;

long long sumOfSeries(int n)
{

    return long(n) * (long)(n + 1) / 2;
}

int main()
{
    int n;
    cout << "Enter the number : " << endl;
    cin >> n;
    cout << "Sum of the series till " << n << " is : " << sumOfSeries(n) << endl;
    return 0;
}

File: day01.java
import java.util.Scanner;

public class day01 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number : ");
        int n = scanner.nextInt();
        System.out.println("The sum of the series is : " + sumSeries(n));
        scanner.close();
    }

    public static long sumSeries(int n) {
        return (long) (n) * (long) (n + 1) / 2;
    }
}